dd-sdk-android/src/main/kotlin/com/datadog/android/log/Logger.kt

```diff
@@ -434,11 +434,41 @@ internal constructor(private val handler: LogHandler) {
 
     // region Internal
 
+    internal fun log(
```

Comment:
Instead of creating these 2 additional methods, you can simply make the existing one `internal` instead of `private`

Reply:
right, good point ;)